pkg/policy/loader.go

```diff
@@ -116,7 +116,10 @@ func (pl *policyLoader) getBuiltInPolicies(ctx context.Context) ([]string, error
 	if err = client.DownloadBuiltinChecks(ctx, pl.RegistryOptions); err != nil {
 		return nil, xerrors.Errorf("failed to download built-in policies: %w", err)
 	}
-	return client.LoadBuiltinChecks()
+	var paths []string
+	paths = append(paths, client.LoadBuiltinChecks())
+
+	return paths, nil
```

Comment:
```suggestion
	return []string{ client.LoadBuiltinChecks() }, nil
```

Reply:
This change doesn't matter, but it's more readable for me

Reply:
oh nice - yeah I will update that.